docs/ocis/getting-started.md

```diff
@@ -91,9 +91,10 @@ marie:radioactivity
 richard:superfluidity
 ```
 
-There is an admin demo account:
+There are admin demo accounts:
 ```console
 moss:vista
+admin:admin
```

Comment:
Note: in the core test suite it is possible to define the environment variables:
`ADMIN_USERNAME`
`ADMIN_PASSWORD`

and the API tests should use those as the authentication for all admin test steps.

Setting those to `moss` and `vista` should work.

But it won't hurt to have an `admin:admin` account existing.